static/src/js/components/SpatialDisplay/__tests__/SpatialDisplay.test.js

```diff
@@ -205,406 +196,537 @@ describe('SpatialDisplay component', () => {
-      const { enzymeWrapper } = setup()
+    // Not sure how to reproduce thist test in RTL.
+    // perhaps checking that on focus and clearing
+    // test.skip('focusing the bounding box NE field sets the manuallyEntering state', () => {
+    //   const { enzymeWrapper } = setup()
 
-      const newBoundingBox = '-77.119759,38.791645,-76.909393,38.995845' // Lon,Lat,Lon,Lat
-      enzymeWrapper.setProps({ boundingBoxSearch: [newBoundingBox] })
+    //   const newBoundingBox = '-77.119759,38.791645,-76.909393,38.995845' // Lon,Lat,Lon,Lat
+    //   enzymeWrapper.setProps({ boundingBoxSearch: [newBoundingBox] })
 
-      const filterStackContents = enzymeWrapper.find(FilterStackContents)
-      const ne = filterStackContents.props().body.props.children.props.children[1]
-      const neInput = ne.props.children[1].props.children
+    //   const filterStackContents = enzymeWrapper.find(FilterStackContents)
+    //   const ne = filterStackContents.props().body.props.children.props.children[1]
+    //   const neInput = ne.props.children[1].props.children
 
-      neInput.props.onFocus()
+    //   neInput.props.onFocus()
 
-      expect(enzymeWrapper.state().manuallyEntering).toEqual('rectangle')
-    })
+    //   expect(enzymeWrapper.state().manuallyEntering).toEqual('rectangle')
+    // })
 
-    test('focusing the circle center field sets the manuallyEntering state', () => {
-      const { enzymeWrapper } = setup()
+    // Not sure how to reproduce thist test in RTL.
+    // perhaps checking that on focus and clearing
+    // test.skip('focusing the circle center field sets the manuallyEntering state', () => {
+    //   const { enzymeWrapper } = setup()
 
-      const newCircle = '-77.119759,38.791645,20000'
-      enzymeWrapper.setProps({ circleSearch: [newCircle] })
+    //   const newCircle = '-77.119759,38.791645,20000'
+    //   enzymeWrapper.setProps({ circleSearch: [newCircle] })
 
-      const filterStackContents = enzymeWrapper.find(FilterStackContents)
-      const center = filterStackContents.props().body.props.children.props.children[0]
-      const centerInput = center.props.children[1].props.children
+    //   const filterStackContents = enzymeWrapper.find(FilterStackContents)
+    //   const center = filterStackContents.props().body.props.children.props.children[0]
+    //   const centerInput = center.props.children[1].props.children
 
-      centerInput.props.onFocus()
+    //   centerInput.props.onFocus()
 
-      expect(enzymeWrapper.state().manuallyEntering).toEqual('circle')
-    })
+    //   expect(enzymeWrapper.state().manuallyEntering).toEqual('circle')
+    // })
 
-    test('focusing the circle radius field sets the manuallyEntering state', () => {
-      const { enzymeWrapper } = setup()
+    // Not sure how to reproduce thist test in RTL.
+    // perhaps checking that on focus and clearing
+    // test.skip('focusing the circle radius field sets the manuallyEntering state', () => {
```

Comment:
Yea looking at these I agree that they are being covered by other tests and aren't in line with how RTL testing is supposed to work making them ill suited for migrating to it